pkg/webui/console/containers/join-eui-prefixes-input/join-eui-prefixes-input.js

```diff
@@ -178,6 +184,13 @@ class JoinEUIPrefixesInput extends React.PureComponent {
           onChange={this.handleChange}
           onBlur={this.handleBlur}
           error={error}
+          action={{
+            icon: 'autorenew',
+            type: 'button',
+            title: m.zeroInput,
+            onClick: this.handleZerosClick,
+            raw: true,
+          }}
```

Comment:
```suggestion
          action={{
            type: 'button',
            title: m.zeroInput,
            onClick: this.handleZerosClick,
            raw: true,
            children: (<span className={style.zeroFillButton}>00</span>)
          }}
```

We still need to use the `00` "icon".

Reply:
Sorry, wasn't clear enough on that one.